dlls/winevulkan/vulkan.c

```diff
@@ -2244,7 +2203,7 @@ static VkResult init_fs_hack_images(VkDevice device, struct VkSwapchainKHR_T *sw
         viewInfo.sType = VK_STRUCTURE_TYPE_IMAGE_VIEW_CREATE_INFO;
         viewInfo.image = swapchain->fs_hack_images[i].user_image;
         viewInfo.viewType = VK_IMAGE_VIEW_TYPE_2D;
-        viewInfo.format = createinfo->imageFormat;
+        viewInfo.format = VK_FORMAT_B8G8R8A8_SRGB;
```

Comment:
What if the app requests something like RGB10A2 or RGBA16F as the swap chain format? Does fshack filter the supported swap chain formats anywhere?

Reply:
No, but we have a big `FIXME("swapchain image format is not BGRA8 UNORM/SRGB. Things may go badly. %d\n", native_info.imageFormat);`. I doubt we will ever see anything but BGRA8 on X11.